adoc/vi-Physical/SA.adoc

```diff
@@ -1340,37 +1373,37 @@ states
 ----
 
 The main difference to the HANA and Netweaver formula is that the _init.sls_ already makes use of _jinja_.
-Jinja is the default templating language in SLS files and get evaluated before YAML, which means it is evaluated before the States are run.
+Jinja is the default templating language in SLS files and gets evaluated before YAML, which means it is evaluated before the states are run.
 
 The most basic usage of Jinja in state files is using control structures to wrap conditional or redundant state elements.
 
 
 ==== Additional Services
 
-The additinal services depend on what is used or available from the cloud provider, but needed by SAP HANA or SAP Netweaver or the HA services.
+The additional services depend on what is used or available from the cloud provider, but needed by SAP HANA or SAP Netweaver or the HA services.
 
 ifeval::[ "{cloud}" == "Azure" ]
 
 ===== NFS service
 
-To build an HA-NFS service, we use the above described _habootstrap-formula_ together with _drbd-formula_ to mirror the data between two nodes and the _linux nfs-server: packages been setup with the saltstack _nfs_formula ( see https://github.com/saltstack-formulas/nfs-formula )
+To build an HA-NFS service, we use the above described _habootstrap-formula_ together with _drbd-formula_ to mirror the data between two nodes and the _linux nfs-server: packages been setup with the SaltStack _nfs_formula ( see https://github.com/saltstack-formulas/nfs-formula )
 
-DRBD®– software is a distributed replicated storage system for the Linux platform. It is implemented as a kernel driver, several userspace management applications, and some shell scripts. So simplified, think about it as an raid-1 over network.
+DRBD®– software is a distributed replicated storage system for the Linux atform. It is implemented as a kernel driver, several userspace management applications, and some shell scripts. So think about it as "RAID-1 over network."
 
 Details are available at the SUSE documentation page for the SLE HA Extension
 https://documentation.suse.com/sle-ha/15-SP2/single-html/SLE-HA-nfs-quick/#art-sleha-nfs-quick
 
 ===== Fencing service
 
-If the setup is using HA for SAP Netweaver or SAP HANA or with the NFS service, and there is mechanism for fencing of the virtual machines over an API we use the SUSE method of using a SBD-device. Such a SBD-Device is normally a raw shared disk beween two nodes.
+If the setup is using HA for SAP Netweaver or SAP HANA with the NFS service, and there is mechanism for fencing of the virtual machines over an API we use the SUSE method of using a SBD-device. Such a SBD-Device is normally a raw shared disk beween two nodes.
```

Comment:
This is now wrong. We have 3 clusters which need fencing, HA HANA, HA Netweaver, HA NFS